pages/authentication/auth.css

```diff
@@ -0,0 +1,50 @@
+@import url(../../basic.css);
+.auth-container {
+    margin-top: 4rem;
+    display: flex;
+    justify-content: center;
+}
+
+form {
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+    padding: 2rem 2rem;
+    background-color: var(--white);
+}
+
+form a {
+    text-decoration: none;
+    color: var(--black);
+}
+
+.forgot-password {
+    font-size: 0.9rem;
+    float: right;
+}
+
+.authentication-btn {
+    display: block;
+    width: 100%;
+    margin: 1rem 0;
+    padding: 0.5rem;
```

Comment:
You can create css variables for spacing, so that it remains consistent everywhere.